ui/app/AppLayouts/stores/ContactsStore.qml

```diff
@@ -16,6 +17,8 @@ QtObject {
         readonly property var contactsModuleInst: profileSectionModule.contactsModule
         readonly property var mainModuleInst: mainModule
         readonly property var globalUtilsInst: globalUtils
+        readonly property var communitiesModuleInst: Global.appIsReady ? communitiesModule : null
```

Comment:
Ideally I think we should not access (or newly introduce) this Global singleton access here